code/modules/mob/living/simple_animal/hostile/megafauna/hierophant.dm

```diff
@@ -399,6 +453,8 @@ Difficulty: Hard
 	if(health > 0 || stat == DEAD)
 		return
 	else
+		for(var/mob/living/simple_animal/hostile/megafauna/colossus/C in GLOB.mob_list)
+			UnregisterSignal(C, COMSIG_MOB_DEATH)
```

Comment:
Shouldnt this be COMSIG_MOB_APPLY_DAMAGE?
```suggestion
			UnregisterSignal(C, COMSIG_MOB_APPLY_DAMAGE)
```

Reply:
ah shit, yeah

Reply:
done